conans/client/tools/system_pm.py

```diff
@@ -114,6 +116,18 @@ def install(self, packages, update=True, force=False):
             self.update()
         self._install_any(packages)
 
+    def _parse_packages_arch(self, packages, platforms):
```

Comment:
Not a good function name either.

Reply:
I think it would be better to have a function (could be on the base class, reading the dicts from subclasses) to get the "suffix" corresponding to the "current" architecture. Then a tiny method `get_package_name(name, suffix)` in the needed subclasses.

Reply:
done